app/common/src/map/DropMarker.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+
+import Color from '../Color';
+
+type Props = {|
+  size?: number,
+|};
+
+/**
+ * This drop marker is always pointing to the (0,0) coordinate. It's because
+ * the marker itself is always absolutely shifted to the left-top corner as
+ * shown on the following picture:
+ *
+ *  .-.
+ *  \O/
+ *   v
+ *   .-------.
+ *   |       |
+ *   |   x   |
+ *   |       |
+ *   `-------`
+ */
+export default function DropMarker({ size = 50 }: Props) {
+  return (
+    <View style={{ position: 'relative' }}>
```

Comment:
isn't `position: 'relative'` set by default in RN?

Reply:
Yes, yes it is. I did it for a reason that is funny and it will never happen so I will rather remove it...

Reply:
Ok, should be fixed now.